compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Numeric.hs

```diff
@@ -3,7 +3,7 @@
 
 -- | DAML-LF Numeric literals, with scale attached.
 module DA.Daml.LF.Ast.Numeric
-    ( Numeric
+    ( Numeric (..)
```

Comment:
I don't think this is a good idea. Why did you change this?

Reply:
Actually, this is fine. I cannot read and thought we were changing a DAML file here, which we obviously aren't. So it's all good. Apologies for the confusion I caused.